services/m3dbnode/server/server.go

```diff
@@ -790,3 +835,60 @@ func capacityPoolOptions(
 	}
 	return opts
 }
+
+func createStaticShardSet(numShards int, listenAddress string) (sharding.ShardSet, []topology.HostShardSet, error) {
+	var (
+		shardSet      sharding.ShardSet
+		hostShardSets []topology.HostShardSet
+		shardIDs      []uint32
+		err           error
+	)
+
+	for i := uint32(0); i < uint32(numShards); i++ {
+		shardIDs = append(shardIDs, i)
+	}
+
+	shards := sharding.NewShards(shardIDs, shard.Available)
+	shardSet, err = sharding.NewShardSet(shards, sharding.DefaultHashFn(1))
+	if err != nil {
+		return nil, nil, err
+	}
+
+	host := topology.NewHost("localhost", listenAddress)
+	hostShardSet := topology.NewHostShardSet(host, shardSet)
+	hostShardSets = append(hostShardSets, hostShardSet)
+
+	return shardSet, hostShardSets, nil
+}
+
+func createMetaData(cfg config.StaticNamespaceConfiguration) (namespace.Metadata, error) {
```

Comment:
nit: "metadata" is one word so this should be `createMetadata` also probably better to be specific and mention the word "namespace" in the method name.  Another thing is by default all our "constructor-like" functions are prefixed with "new" so for consistency we should probably apply the same rule here. 
 `newNamespaceMetadata(...)` seems to incorporate all that as a name.